controller/src/lib.rs

```diff
@@ -632,6 +747,215 @@ impl Controller {
         }
     }
 
+    /// Set the power mode for a set of transceiver modules.
+    ///
+    /// This method may be used regardless of whether a module uses hardware
+    /// control or software override for controlling the power.
+    pub async fn set_power_mode(&self, modules: ModuleId, mode: PowerMode) -> Result<(), Error> {
+        // How we proceed largely depends on two things: whether we're turning
+        // the power OFF entirely, and whether a module has set software
+        // override of the `LPMode` pin.
+        match mode {
+            PowerMode::Off => {
+                self.assert_lpmode(modules).await?;
```

Comment:
While the module is powered off we actually want `LpMode` to be deasserted (low). The out of date RFD 244 has [the context](https://rfd.shared.oxide.computer/rfd/0244#_fpga_module_sequencing) which still holds:
>While we want to assert LPMode as soon as possible, we cannot do so until the module has had power enabled, otherwise the modules bleed out onto their main voltage rail. This is documented in [hardware-qsfp-x32#47](https://github.com/oxidecomputer/hardware-qsfp-x32/issues/47)

Reply:
Ok, thanks for the context. I'll change that and add a link.

So I understand, does this mean that it's possible for a module to turn on and go directly to high-power mode? The code in this PR always turns on the power then sets `LPMode`, but those could be separated by a while since this is all over the network. What happens in that case?